changelog/covopts.dd

```diff
@@ -0,0 +1,7 @@
+The runtime learned a few new options to customize how coverage reports are created:
+
+$(DL
+  $(DT covmerge) $(DD Set merge mode; either N/n/0 or Y/y/1.)
+  $(DT covsrcpath) $(DD Set path to where source files are located.)
+  $(DT covdstpath) $(DD Set path to where listing files are to be written.)
```

Comment:
I would make the usage a bit more explicit. E.g. first line:
>  $(DT --DRT-covmerge=Y) $(DD Set merge mode; Accepts N/n/0 or Y/y/1. Default to Y.)

Also for this specific part (covmerge), it might be more consistent to use {false,0} / {true,1}

Reply:
I'm copying how `gcopt` parse options (more or less).

Reply:
I was just about to try to generalize how the `gc` parse the options, but it was a much bigger task and I'm not sure I fully understand the restrictions on druntime to do it properly (for example, I don't know why this `rt_configOption` is not just imported normally instead of using this `externDFunc` thing), and then I didn't know if it will get accepted.

At some point I think it would be nice to have a generalized option parsing mechanism like the one in `src/gc/config.d`.